Doc/library/typing.rst

```diff
@@ -623,14 +623,21 @@ These can be used as types in annotations using ``[]``, each having a unique syn
 
 .. data:: Union
 
-   Union type; ``Union[X, Y]`` is equivalent to ``X | Y`` and means either X or Y.
+   Union type is defined as ``X | Y`` and means either X or Y.
 
-   To define a union, use e.g. ``Union[int, str]`` or the shorthand ``int | str``.  Details:
+   There's also an older version ``Union[int, str]``.
+   Note that the ``X | Y`` is preferred in readability in most cases.
+   However, there are :ref:`specified runtime differences<types-union>`.
```

Comment:
is this linking to https://docs.python.org/3.11/library/stdtypes.html#types-union ?

The wording there doesn't specifically cover Union vs | behavior differences.